next.config.js

```diff
@@ -4,7 +4,9 @@ const nextConfig = {
   swcMinify: true,
   images: {
     dangerouslyAllowSVG: true,
-    domains: ['user-images.githubusercontent.com', 'api.producthunt.com']
+    domains: ['user-images.githubusercontent.com', 'api.producthunt.com', 'www.shapedivider.app',
```

Comment:
We can remove('www.shapedivider.app', 'haikei.app') and host the image on GitHub(attached in an issue.)

Reply:
@clebsonf Will remove. Shall I create the issue or will you?